mail/mpack/Portfile

```diff
@@ -11,14 +11,16 @@ license			BSD
 description     	mime mail packing and unpacking
 long_description	mpack and munpack provide command line tools to \
 			pack and unpack multipart mime mail messages
-platforms       	darwin
 categories      	mail
 maintainers     	nomaintainer
 
 homepage		https://en.wikipedia.org/wiki/Mpack_(Unix)
 master_sites		macports_distfiles
 
 patchfiles		implicit.patch \
+			makefile.am.patch \
 			patch-TMPDIR.diff
 
 configure.args  	--mandir=${prefix}/share/man
```

Comment:
`${prefix}/share/man` is the default value for `mandir` in recent versions of autotools. Now that you're autoreconfing, there's no need to specify this.

```suggestion
```